includes/definitions/discovery/nios.yaml

```diff
@@ -18,4 +18,37 @@ modules:
                     descr: HA Status
                     states:
                         - { value: 1, generic: 0, graph: 1, descr: Active }
-                        - { value: 2, generic: 0, graph: 1, descr: Passive }
\ No newline at end of file
+                        - { value: 2, generic: 0, graph: 1, descr: Passive }
+                -
+                    oid: ibServiceStatus
+                    num_oid: '.1.3.6.1.4.1.7779.3.1.1.2.1.9.1.2.1'
+                    index: 'ibServiceName.1'
```

Comment:
the 2 ".1" ending those 2 lines up this comment should use {{ $index }} to make it dynamic for all entries of the table. And the service name has to be taken from another raw in the snmp tree. 

try this : 
oid: ibMemberServiceStatusEntry
value: ibServiceStatus
descr: 'Service {{ $ibServiceName }} Status'

Reply:
I can make the discovery work, however whatever I have tried, the first poll immediately changes all the values to zero. I'm guessing that the index that is written to the db is not numeric, but I don't actually know which table to query.....

                -
                    oid: ibMemberServiceStatusEntry
                    value: ibServiceStatus
                    num_oid: '.1.3.6.1.4.1.7779.3.1.1.2.1.9.1.{{ $index }}'
                    index: 'ibMemberServiceStatusEntry.{{ $index }}'
                    descr: '{{ $ibServiceDesc }}'
                    states:
                        - { value: 1, generic: 0, graph: 0, descr: Working }
                        - { value: 2, generic: 1, graph: 0, descr: Warning }
                        - { value: 3, generic: 2, graph: 0, descr: Failed }
                        - { value: 4, generic: 2, graph: 0, descr: Inactive }
                        - { value: 5, generic: 3, graph: 0, descr: Unknown }

Reply:
The snmpwalk in polling gives this:

.1.3.6.1.4.1.7779.3.1.1.2.1.9.1.2.dhcp: Unknown Object Identifier (dhcp)
.1.3.6.1.4.1.7779.3.1.1.2.1.9.1.2.dns: Unknown Object Identifier (dns)
.1.3.6.1.4.1.7779.3.1.1.2.1.9.1.2.ntp: Unknown Object Identifier (ntp)
.1.3.6.1.4.1.7779.3.1.1.2.1.9.1.2.tftp: Unknown Object Identifier (tftp)
.1.3.6.1.4.1.7779.3.1.1.2.1.9.1.2.http-file-dist: Unknown Object Identifier (http-file-dist)
.1.3.6.1.4.1.7779.3.1.1.2.1.9.1.2.ftp: Unknown Object Identifier (ftp)
.1.3.6.1.4.1.7779.3.1.1.2.1.9.1.2.bloxtools-move: Unknown Object Identifier (bloxtools-move)
.1.3.6.1.4.1.7779.3.1.1.2.1.9.1.2.bloxtools: Unknown Object Identifier (bloxtools)

Reply:
They use string index, not numerical indexes. 
https://docs.librenms.org/Developing/os/Health-Information/

```
num_oid (required): This is the numerical OID that contains value. This should always include 
{{ $index }}. snmptranslate -On can help figure out the number. In case the index is a string,
 {{ $index_string }} can be used instead.
```

Reply:
https://p.libren.ms/view/0856981e
https://p.libren.ms/view/ad2df3a7
the snmpbulkwalk fails.....
<h4>A PHP Error was encountered</h4>
<p>Severity: Warning</p>
<p>Message:  Error while sending QUERY packet. PID=3148</p>
<p>Filename: mysqli/mysqli_driver.php</p>
<p>Line Number: 305</p>